ARC-0071: Soulbound ASA NFT

SudoWeezy opened the pull request:
Another version of https://github.com/algorandfoundation/ARCs/pull/134
Based on the ERC-5484: https://eips.ethereum.org/EIPS/eip-5484

SudoWeezy committed:
New version of soulbound ASA

SudoWeezy committed:
update link

SudoWeezy committed:
Update arc-0071.md

SudoWeezy committed:
Update arc-0071.md

SudoWeezy committed:
Remove template sections

SudoWeezy committed:
Update arc-0071.md

SudoWeezy committed:
Update arc-0071.md

SudoWeezy committed:
Update arc-0071.md

SudoWeezy committed:
Update arc-0071.md

SudoWeezy committed:
Fix format ARC

SudoWeezy committed:
Update arc-0071.md

JuWeber99 commented:
As AVM9 as far as I read plans to introduce primitives for elliptic curve pairings I whould suggest mentioning or working out more on the relation that can be established between an SBT and zero knowledge proof to establish private on chain verification possibilities that whould be useful for loan records and several other usecases mentioned. Greetings :)

emg110 commented:
Amazing work @SudoWeezy ! This is going to be needed in many applications and use cases : DIDs and VCs being one of them and GoPlausible will dedicate a special focus of resources on ARC71 from now on to help take it forward! Thanks again!

SudoWeezy commented:
Closing in favor of https://github.com/algorandfoundation/ARCs/pull/242

emg110 commented:
This is so back on!

emg110 committed:
ARC-0071 Consensual Soul Bound ASA

emg110 committed:
Fix of flaw in design

emg110 committed:
Minor fix

emg110 committed:
Contributions

emg110 committed:
TODO

emg110 committed:
Find another flaw!

emg110 committed:
Fix Typo

emg110 committed:
Update (address some flaws)

emg110 committed:
Add Noted Issues

emg110 committed:
Update

emg110 committed:
Add active solution for closeout issue

emg110 committed:
Format

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Change requirements

emg110 committed:
Update requirements

emg110 committed:
Update Solutions

emg110 committed:
Update second approach

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update wordings

emg110 committed:
Update: solution 2 eliminated

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Final Update

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update Security Considerations

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update

emg110 committed:
Update content

emg110 committed:
Update by Stephane

emg110 committed:
Fix Typo ARC19

emg110 committed:
Update

emg110 committed:
Update

SudoWeezy committed:
Merge pull request #316 from GoPlausible/ARC-71

ARC-0071 Consensual Soul Bound ASA [New Approach]

emg110 commented:
After some overhaul and applying some changes, a new ready-for-review version has become ready.
Please read [this Gist article](https://gist.github.com/emg110/775d706bdbfed716d417f998b9e8c999), for more info.
A complete reference implementation on the production level has been made ready under @GoPlausible. 
OpenSource reference implementation will be added soon using TEAL and SDK.

emg110 committed:
Merge pull request #1 from algorandfoundation/ARC-71

Merge pull request #316 from GoPlausible/ARC-71

emg110 committed:
Fixed sentence mistake and corrected to "The Freeze parameter **MUST** be set to the Issuer Address"

SudoWeezy committed:
Merge pull request #323 from GoPlausible/ARC-71

Arc 71- Fixed a text mistake

SudoWeezy committed:
Manager parameter **MAY** be set to any address

- The Manager parameter **MAY** be set to any address but is **RECOMMENDED** to be the Issuer.

SudoWeezy committed:
Update arc-0071.md
